Coordinator/Coordinator/App/View/LoginView.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  LoginView.swift
+//  Coordinator
+//
+//  Created by Developer on 01.10.2021.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+final class LoginView: UIView {
+
+    //MARK: - Visual Elements
+    let loginTextField: UITextField = {
+        let textField = UITextField()
+        textField.placeholder = " Login"
+        textField.layer.borderWidth = 0.5
+        textField.layer.cornerRadius = 9
+        textField.alpha = 0.5
+        return textField
+    }()
+
+    let passwordTextField: UITextField = {
+        let textField = UITextField()
+        textField.placeholder = " Password"
+        textField.layer.borderWidth = 0.5
+        textField.layer.cornerRadius = 9
+        textField.alpha = 0.5
+        return textField
+    }()
+
+    let authButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("Auth", for: .normal)
+        button.setTitle("No", for: .disabled)
+        button.setTitleColor(.black, for: .disabled)
+        button.layer.cornerRadius = 8
+        button.addTarget(self, action: #selector(toFirstVC), for: .touchUpInside)
+        return button
+    }()
+
+    //MARK: - Internal Properties
+    var completion: (()->())?
+
+    //MARK: - Initializers
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupView()
+        setupLoginTextField()
+        setupPasswordTextField()
+        setupAuthButton()
+        bindViewModel()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setupView()
+        setupLoginTextField()
+        setupPasswordTextField()
+        setupAuthButton()
+        bindViewModel()
+    }
+
+    //MARK: - Private Methods
+    @objc private func toFirstVC() {
```

Comment:
```suggestion
    @objc private func onTap() {
```
не должна вью знать куда переходим говорит когда нажали